api/src/main/java/org/pmiops/workbench/cohortreview/AnnotationQueryBuilder.java

```diff
@@ -137,10 +137,8 @@ private String getWhereSql(CohortReview cohortReview, List<CohortStatus> statusF
       ImmutableMap.Builder<String, Object> parameters) {
     StringBuilder whereBuilder = new StringBuilder(" WHERE pcs.cohort_review_id = :cohort_review_id");
     parameters.put("cohort_review_id", cohortReview.getCohortReviewId());
-    if (!statusFilter.containsAll(REVIEWED_STATUSES)) {
-      whereBuilder.append(" AND pcs.status IN (:statuses)");
-      parameters.put("statuses", statusFilter.stream().map(CohortStatus::ordinal).collect(Collectors.toList()));
-    }
+    whereBuilder.append(" AND pcs.status IN (:statuses)");
```

Comment:
Existing issue, but this library depends on statusFilter being non-empty, so should remove that precondition or check for it explicitly. Granted it will work for now since there's only one client which always sends a non-empty list.

Reply:
Done.